lib/pdksync.rb

```diff
@@ -207,6 +218,60 @@ def self.main(steps: [:clone], args: nil)
         Utils.delete_branch(client, repo_name, module_args[:branch_name])
         PdkSync::Logger.info 'branch deleted'
       end
+
+      if steps.include?(:run_tests_jenkins)
+        Dir.chdir(main_path) unless Dir.pwd == main_path
+        print 'run tests in jenkins, '
+        module_type = Utils.module_type(output_path, module_name)
+        if module_type == 'traditional'
+          github_user = 'puppetlabs' if module_args[:test_framework].nil?
+          github_user = module_args[:github_user] unless module_args[:github_user].nil?
+          if module_args[:test_framework] == 'jenkins' || module_args[:test_framework].nil?
+            module_name = "puppetlabs-#{module_name}" if %w[cisco_ios device_manager].include?(module_name) # rubocop:disable Metrics/BlockNesting
```

Comment:
can we remove the hardcoded module names

Reply:
Hey @sheenaajay ,

No, not for these particular cases, because they don't follow the same syntax as the others
e.g.
repo name: https://github.com/puppetlabs/device_manager
most of them have something like: https://github.com/puppetlabs/puppetlabs-java_ks
Cheers!